config/routes.rb

```diff
@@ -72,9 +72,18 @@
     end
   end
 
-  resources :collections, concerns: [:collaboratable, :activities]
+  resources :collections, concerns: %i[collaboratable activities] do
+    member do
+      %w[events materials].each do |item|
+        return unless TeSS::Config.feature[item]
```

Comment:
Better to check feature in controller: https://github.com/ElixirTeSS/TeSS/issues/729